ArduCopter/commands_logic.cpp

```diff
@@ -651,6 +655,29 @@ void Copter::do_guided_limits(const AP_Mission::Mission_Command& cmd)
 }
 #endif
 
+// control winch based on mission command
+void Copter::do_winch(const AP_Mission::Mission_Command& cmd)
+{
+    // Note: we ignore the gripper num parameter because we only support one gripper
+    switch (cmd.content.winch.action) {
+        case WINCH_DISABLE:
+            g2.winch.relax();
+            Log_Write_Event(DATA_WINCH_DISABLE);
```

Comment:
Out of scope for this PR, so no need to address, but it would be nice if one day the Log_Write_Event stuff lived in Dataflash so that the winch library could automatically stick the events in.